tests/contrib/gunicorn/post_fork.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+import gevent.monkey
+
+from ddtrace.debugging import DynamicInstrumentation
+from ddtrace.internal.remoteconfig import RemoteConfig
+from ddtrace.profiling.profiler import Profiler
+
+
+# take some notes about the relative ordering of thread creation and
+# monkeypatching
+monkeypatch_happened = gevent.monkey.is_module_patched("threading")
+
+# enabling DI here allows test cases to exercise the code paths that handle
+# gevent monkeypatching of running threads
+# post_fork is called before gevent.monkey.patch_all()
+if sys.version_info[1] < 11:
```

Comment:
We can future proof this check:

```suggestion
if sys.version_info < (3, 11):
```

Reply:
We should remove this check after DynamicInstrumentation add support for python 3.11. Maybe we should leave comment here?

Reply:
I think a recursive grep for `(3, 11)` will be enough to track this down once the DI change is merged.